### PR TITLE
fmdiff: modernize

### PR DESCRIPTION
**Pull request by bfontaine**
I can’t find a way to write a test, all tools have a GUI and no `--help`/`--version` flags.

### CODE REVIEW

**Comment by vszakats** [comment]
`homepage` accepts https, would you mind updating it?

**Comment by bfontaine** [comment]
Fixed, thanks!

File: Library/Formula/fmdiff.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-  sha1 'a5342820893f61b29e1060527cc722ef51574911'
+  homepage "https://www.defraine.net/~brunod/fmdiff/"
+  url "http://bruno.defraine.net/fmdiff/fmscripts-20120813.tar.gz"
+  sha256 "7312654040acc29787c15fb2b6d53abe0a397fb9faec6bf43398d25bb31f38ee"
 
-  head 'http://soft.vub.ac.be/svn-gen/bdefrain/fmscripts/', :using => :svn
+  head "http://soft.vub.ac.be/svn-gen/bdefrain/fmscripts/", :using => :svn
 
   def install
   system "make"
```
Comment (mikemcquaid) [inline Library/Formula/fmdiff.rb]: Mind fixing the indentation in this function?

**Comment by mikemcquaid** [comment]
> I can’t find a way to write a test, all tools have a GUI and no --help/--version flags.

Probably just do a `File.exist?` type test?

**Comment by mikemcquaid** [comment]
A hacky test looks like you could set PATH and write a dummy opendiff script and check it gets called by these scripts?

**Comment by bfontaine** [comment]
> A hacky test looks like you could set PATH and write a dummy opendiff script and check it gets called by these scripts?

Yep, just added a hacky test with a dummy `filemerge` script.

**Comment by bfontaine** [comment]
Is this test ok @mikemcquaid?

**Comment by mikemcquaid** [comment]
@bfontaine Yup but if you could fix the indentation too that'd be :heart:

File: Library/Formula/fmdiff.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/fmdiff.rb]: Sorry I missed your comment, fixed!

**Pull request closed without merging**